Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -1337,6 +1337,7 @@
             "use_ip_nt_range_tags_include", "use_ip_nt_range_tags_exclude",
             "active",
             "scanners_in_network",
+            "fqdn",
```

Comment:
why did you add it

Reply:
I added it because Qualysv2 lacked the parameter fqdn
If not added, Demisto returns an error:

![scan_create_no_fqdn](https://github.com/demisto/content/assets/134506710/f3465921-2545-4a9d-a293-fa4ece44808a)